test/e2e/plain_provisioner_test.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
 	plain "github.com/operator-framework/rukpak/internal/provisioner/plain/types"
+	registry "github.com/operator-framework/rukpak/internal/provisioner/registry/types"
```

Comment:
Should this be inside of a new file? Something like:

`registry_provisioner_test.go`

Reply:
I make the new file.  It was just one test so I thought a new file might not necessary but we probably add more tests for the registry bundle so I create the new file.  Thanks!